ckanext/datatablesview/plugin.py

```diff
@@ -32,6 +32,15 @@ def can_view(self, data_dict):
         return resource.get(u'datastore_active')
 
     def view_template(self, context, data_dict):
+        '''
+        Set the index of the sort column if it's displayed
+        '''
+        resource_view = data_dict.get('resource_view')
+        sort_column = resource_view.get('sort_column')
+        show_fields = resource_view.get('show_fields', [])
+        if sort_column in show_fields:
```

Comment:
I am asking this question for my own understanding (sort_column not in show_fields:). If the sort_column is hidden, what happens? 
Are we preventing users from sorting on invisible columns?

Reply:
Users will only be able to sort on visible columns.